pkg/migration/migration.go

```diff
@@ -33,14 +32,8 @@ func (m *Migration) Run() error {
 		return err
 	}
 	defer migration.Close()
-	if !m.SkipPreRunChecks {
```

Comment:
This change doesn't seem related to adding progress?

Reply:
No, it's removing 5.7 specific dead code I caught. See the comment just below.